src/components/ConnectButton.vue

```diff
@@ -219,7 +219,7 @@ export default {
 <template lang='pug'>
 div()
     q-btn( v-if='!isLoggedIn' id='c-connect-button__login-button' label='Connect Wallet' @click='connect()' )
-    q-btn-dropdown( v-if='isLoggedIn' :label='getLoginDisplay()' )
+    q-btn-dropdown.q-ml-sm(flat round v-if='isLoggedIn' :label='getLoginDisplay()' )
```

Comment:
there is now a small part of the button on the left which is not clickable and when you hover, nothing happens. I think switching this to `.q-pl-sm` might fix it

Reply:
Actually that doesn't work, no padding is applied. Best is to remove that margin an call it a day I think.